packages/ckeditor5-dev-webpack-plugin/lib/servetranslations.js

```diff
@@ -54,33 +54,38 @@ module.exports = function serveTranslations( compiler, options, translationServi
 		}
 	}
 
-	// Add core translations before `translatesourceloader` starts translating.
-	compiler.plugin( 'after-resolvers', () => {
-		const resolver = compiler.resolvers.normal;
+	// Add core translations before `translateSourceLoader` starts translating.
+	compiler.hooks.normalModuleFactory.tap( 'CKEditor5Plugin', normalModuleFactory => {
```

Comment:
Is this consistent with webpack plugins' naming convention?

Reply:
Yep, see https://github.com/webpack/tapable.